api/MessagePort.json

```diff
@@ -0,0 +1,324 @@
+{
+  "api": {
+    "MessagePort": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MessagePort",
+        "support": {
+          "webview_android": {
+            "version_added": "4"
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": "4"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": [
+            {
+              "version_added": true,
+              "version_removed": "41"
+            },
+            {
+              "version_added": "41",
+              "notes": "Available in workers."
+            }
+          ],
+          "firefox_android": [
+            {
+              "version_added": true,
+              "version_removed": "41"
+            },
+            {
+              "version_added": "41",
+              "notes": "Available in workers."
+            }
+          ],
+          "ie": {
+            "version_added": "10"
+          },
+          "opera": {
+            "version_added": "10.6"
+          },
+          "opera_android": {
+            "version_added": "11.5"
+          },
+          "safari": {
+            "version_added": "5"
+          },
+          "safari_ios": {
+            "version_added": "5.1"
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "postMessage": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MessagePort/postMessage",
+          "support": {
+            "webview_android": {
+              "version_added": "4"
+            },
+            "chrome": {
+              "version_added": "4"
+            },
+            "chrome_android": {
+              "version_added": "4"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
```

Comment:
Do we want to add the same notes (`Available in workers`) for Firefox 41 as above?